src/main/java/tr/com/srdc/cda2fhir/transform/section/impl/SectionResultDynamic.java

```diff
@@ -0,0 +1,37 @@
+package tr.com.srdc.cda2fhir.transform.section.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Resource;
+
+import tr.com.srdc.cda2fhir.transform.section.ISectionResult;
+import tr.com.srdc.cda2fhir.util.FHIRUtil;
+
+public class SectionResultDynamic implements ISectionResult  {
+	private Bundle bundle = new Bundle();
+	private List<Resource> resources = new ArrayList<Resource>();
+	
+
+	public void add(Resource resource) {
+        bundle.addEntry().setResource(resource);
+		resources.add(resource);
+	}
+	
+	public <T extends Resource> void merge(Bundle bundle, Class<T> clazz) {
+   		FHIRUtil.mergeBundle(bundle, this.bundle);
+   		List<? extends Resource> sectionResources = FHIRUtil.findResources(bundle, clazz);
+		resources.addAll(sectionResources);
```

Comment:
Couldn't this lead to duplicates if called multiple times? Wouldn't we need to clear the current results list?